cmd2/cmd2.py

```diff
@@ -572,18 +569,18 @@ def _register_subcommands(self, cmdset: Union[CommandSet, 'Cmd']) -> None:
 
             for action in command_parser._actions:
                 if isinstance(action, _UnloadableSubParsersAction):
-                    action.add_parser(subcommand_name, parents=[subcmd_parser], help=help_text)
+                    action.add_parser(subcommand_name, parents=[subcmd_parser], help=help_text, aliases=aliases)
```

Comment:
I don't think you should pass `help` and `aliases` if they are `None`.
Look how [argparse](https://github.com/python/cpython/blob/3.8/Lib/argparse.py#L1109) uses these arguments.

I would suggest doing:

```Python
kwargs = dict()
if help_text is not None:
    kwargs['help'] = help_text
if aliases is not None:
    kwargs['aliases '] = aliases

action.add_parser(subcommand_name, parents=[subcmd_parser], **kwargs)
```